react/utils/compatibilityLayer.js

```diff
@@ -63,6 +63,7 @@ export const detachFiltersByType = facets => {
   const groupedFilters = byType(facets)
 
   const brands = pathOr([], ['BRAND', 0, 'facets'], groupedFilters)
+  const brandsQuantity = pathOr(0, ['BRAND', 0, 'quantity'], groupedFilters)
```

Comment:
Remove ramda

```js
  const brandsQuantity = groupedFilters && groupedFilters.BRAND && groupedFilters.BRAND[0] && groupedFilters.BRAND[0].quantity != null ? groupedFilters.BRAND[0].quantity : 0
```
Not using suggestion because linter may want to fix

Reply:
again, can be terser with `?.`

Reply:
it's JS unfortunately